### PR TITLE
Refactoring error and success messages

### PR DESCRIPTION
**Pull request by 16dprice**
I added better error and success messages. I also removed our dependency on jQuery.

---
This change is [Reviewable](https://reviewable.io/reviews/anonymouswalker/safileupload/3)

### CODE REVIEW

File: src/main/resources/templates/index.html [hunk 1]
```diff
@@ -55,9 +51,43 @@
...
     </div>
 
-    <div class="main-grid__output">
-        <label>Output<p class="main-grid__output-text">This is the output</p></label>
-    </div>
+</div>
+
+<div class="main-grid__output main-grid__success">
+    <br>
+    <svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="24"
```
Comment (AnonymousWalker) [inline src/main/resources/templates/index.html]: Should we keep these xmlns namespaces?

Comment (16dprice) [reply src/main/resources/templates/index.html]: I did a bit of research and I don't think they are necessary, actually. I'll remove them.

Reference the top two comments here for why we don't need it: https://stackoverflow.com/questions/18467982/are-svg-parameters-such-as-xmlns-and-version-needed

**Review by jsarabia — CHANGES_REQUESTED** [review]
> *[Reviewable](https://reviewable.io/reviews/anonymouswalker/safileupload/3)* status: 0 of 3 files reviewed, 3 unresolved discussions (waiting on @16dprice, @AnonymousWalker, and @jsarabia)

___
*[src/js/main.js, line 32 at r2](https://reviewable.io/reviews/anonymouswalker/safileupload/3#-MA7xGLk9Y8ieDOfTaVc:-MA7xGLk9Y8ieDOfTaVd:b-hocdqe) ([raw file](https://github.com/anonymouswalker/safileupload/blob/2b218f9e382402d511ecf4b04339d4f4305f50d3/src/js/main.js#L32)):*
> ```JavaScript
>         document.querySelector(".main-grid__success-message").innerHTML = res.data.output
>         document.querySelector(".main-grid__error").classList.remove("main-grid__show-success-status")
>         document.querySelector(".main-grid__success").classList.add("main-grid__show-success-status")
> ```

Why are error and success two different elements? Should this just be a status element, which modifiers for if it's success or error?
___
*[src/js/main.js, line 40 at r2](https://reviewable.io/reviews/anonymouswalker/safileupload/3#-MA7uGqR-j6zoFfciaUw:-MA7uGqU2RDI39fDamRD:b-tynrhd) ([raw file](https://github.com/anonymouswalker/safileupload/blob/2b218f9e382402d511ecf4b04339d4f4305f50d3/src/js/main.js#L40)):*
> ```JavaScript
> 
>     // multiply by 1000 here to convert from seconds to milliseconds
>     removeOutputMessageTimeout = setTimeout(removeOutputMessage, 1000 * timeoutDuration)
> ```

this is rather unnecessary; just make timeout duration 10,000. If you want to make a comment and say "10 seconds" that's fine, or rename it timeoutDurationMs

**Comment by 16dprice** [comment]
___
*[src/js/main.js, line 32 at r2](https://reviewable.io/reviews/anonymouswalker/safileupload/3#-MA7xGLk9Y8ieDOfTaVc:-MA7yhWc5iliLJSHXhHm:b-bffx3x) ([raw file](https://github.com/anonymouswalker/safileupload/blob/2b218f9e382402d511ecf4b04339d4f4305f50d3/src/js/main.js#L32)):*
<details><summary><i>Previously, jsarabia (Joe) wrote…</i></summary><blockquote>

Why are error and success two different elements? Should this just be a status element, which modifiers for if it's success or error?
</blockquote></details>

It made it easier to work with the svg icons that I got from Thomas in order to make error and success messages better. The way you're proposing to do it is the way I originally went about it but I couldn't find a clean way to do it and also update the svg icon and render it. Maybe I'm just missing something though? I believe replacing with innerHTML didn't show the svg, it just showed the actual svg code on the browser

**Comment by 16dprice** [comment]
___
*[src/js/main.js, line 40 at r2](https://reviewable.io/reviews/anonymouswalker/safileupload/3#-MA7uGqR-j6zoFfciaUw:-MA8-Far3z2VdJxCJaNo:b-896fix) ([raw file](https://github.com/anonymouswalker/safileupload/blob/2b218f9e382402d511ecf4b04339d4f4305f50d3/src/js/main.js#L40)):*
<details><summary><i>Previously, jsarabia (Joe) wrote…</i></summary><blockquote>

this is rather unnecessary; just make timeout duration 10,000. If you want to make a comment and say "10 seconds" that's fine, or rename it timeoutDurationMs
</blockquote></details>

Done.

**Comment by 16dprice** [comment]
___
*[src/js/main.js, line 32 at r2](https://reviewable.io/reviews/anonymouswalker/safileupload/3#-MA7xGLk9Y8ieDOfTaVc:-MACYZdw7HzMcmylvCmU:b-896fix) ([raw file](https://github.com/anonymouswalker/safileupload/blob/2b218f9e382402d511ecf4b04339d4f4305f50d3/src/js/main.js#L32)):*
<details><summary><i>Previously, 16dprice wrote…</i></summary><blockquote>

It made it easier to work with the svg icons that I got from Thomas in order to make error and success messages better. The way you're proposing to do it is the way I originally went about it but I couldn't find a clean way to do it and also update the svg icon and render it. Maybe I'm just missing something though? I believe replacing with innerHTML didn't show the svg, it just showed the actual svg code on the browser
</blockquote></details>

Done.

**Comment by 16dprice** [comment]
___
*[src/main/resources/templates/index.html, line 58 at r1](https://reviewable.io/reviews/anonymouswalker/safileupload/3#-MA6_jay8E-u2ruem-Pf-r1-58:-MACYaFI6rMqvt4-t3qT:b-896fix) ([raw file](https://github.com/anonymouswalker/safileupload/blob/c7da77d6c1980db220f3e21c4be00b72886fe3af/src/main/resources/templates/index.html#L58)):*
<details><summary><i>Previously, 16dprice wrote…</i></summary><blockquote>

I did a bit of research and I don't think they are necessary, actually. I'll remove them.

Reference the top two comments here for why we don't need it: https://stackoverflow.com/questions/18467982/are-svg-parameters-such-as-xmlns-and-version-needed
</blockquote></details>

Done.

**Review by jsarabia — APPROVED** [review]
:lgtm:

> *[Reviewable](https://reviewable.io/reviews/anonymouswalker/safileupload/3#-:-MACeKKzC5o1kdxlqRiy:bnfp4nl)* status: 0 of 3 files reviewed, 1 unresolved discussion (waiting on @AnonymousWalker)

**Pull request merged**